tests/test_management_commands.py

```diff
@@ -138,6 +198,17 @@ def check_last_index_actions(
                     self.assertEqual(index_action.action, expected_action, expected_msg)
         return index_actions
 
+    @patch('django_elastic_migrations.management.commands.es_list.log')
+    def check_es_list(self, mock_logger, expected_table, scenario_message, es_only=False, args=None):
```

Comment:
added `check_es_list` test utility which makes it easier to assert that the output of `./manage.py es_list` is as expected.